lib/registries/addon/components/registries-banner/template.hbs

```diff
@@ -1,5 +1,5 @@
 {{#if (and this.stateBanner.text this.stateBanner.type) }}
-    <div local-class='Banner' data-test-banner={{@node.state}}>
+    <div local-class='Banner' data-test-banner={{@registration.state}}>
         <BsAlert @dismissible={{this.dismissible}} @type={{this.stateBanner.type}}>
             <div class='text-center'>
                 {{t this.stateBanner.text embargoEndDate=(moment-format this.registration.embargoEndDate 'MMMM D, YYYY') projectUrl=this.projectUrl}}
```

Comment:
Drive by: these should not be interchangeable. Given that you have a `registration` property in your component, and given that it mutates, is there a reason to use `@registration` rather than `this.registration` everywhere in the template?

Reply:
> given that it mutates

It's not mutated within the component. The only way it might change is if the argument passed from the parent context changes. Given that, `@registration` and `this.registration` *are* interchangeable, since passed-in args are assigned to properties on the component instance. 

The benefit of using `@foo` is that it communicates to devs where the value comes from (the parent context), rather than making them parse `component.ts` to find out.